src/components/Dropdown/Dropdown.tsx

```diff
@@ -14,20 +14,6 @@ const DropdownMenuItem = styled(GenericMenuItem)<{ $type?: "default" | "danger"
   position: relative;
   display: flex;
   min-height: 32px;
-  &[data-state="open"] {
-    ${({ theme, $type = "default" }) => {
-      const colorGroup =
-        theme?.click?.genericMenu?.item?.color?.[$type] ||
-        theme?.click?.genericMenu?.item?.color?.default;
-      if (!colorGroup || !theme?.click?.genericMenu?.item?.typography) return "";
-      return `
-      font: ${theme.click.genericMenu.item.typography.label.hover};
-      background: ${colorGroup.background.hover};
-      color: ${colorGroup.text.hover};
-      cursor: pointer;
-    `;
-    }}
-  }
```

Comment:
The dropdown didn’t have the issue because it had these styles. I’ve moved them to `GenericMenu` so we can reuse them for the context menu and any other components that rely on `GenericMenu`.